### ISSUE TITLE
Missing nodes

### ISSUE THREAD

**Issue by Sindyaev** [kijai/ComfyUI-VEnhancer#8]
All dependencies are installed, but nodes are not displayed
![Uploading 312123123.PNG…]()

**Comment by kijai** [comment]
Then all dependencies simply aren't installed, look for errors in the startup log to see what's missing, should be something like "module not found" or something related to the name of this node pack.

**Comment by yovsac** [comment]
Hello, Im having the same problem. Did you find a solution?

**Comment by janoschsimon** [comment]
did you guys try in the custom folder of ComfyUI-VEnhancer: pip install -r requirements.txt ?

cheers janosch

**Comment by yovsac** [comment]
Hi, not yet. Will check 👍

**Comment by yovsac** [comment]
> did you guys try in the custom folder of ComfyUI-VEnhancer: pip install -r requirements.txt ?
> 
> cheers janosch

I get this error message. do you know why?
![Screenshot 2024-10-28 100049](https://github.com/user-attachments/assets/11296fd5-d842-4171-9400-616f4aa98481)

**Comment by janoschsimon** [comment]
as your in the portable version you need to use the python from x:\..\ComfyUI_windows_portable\python_embeded\python.exe  pip install -r requirements.txt . cheers janosch

**Comment by yovsac** [comment]
Thank you Janosch, but I get this message when I do that. 
![Screenshot 2024-10-28 101745](https://github.com/user-attachments/assets/eae57ff9-af21-4cb4-a5fd-77967ca686be)

**Comment by kijai** [comment]
> > did you guys try in the custom folder of ComfyUI-VEnhancer: pip install -r requirements.txt ?
> > cheers janosch
> 
> I get this error message. do you know why? ![Screenshot 2024-10-28 100049](https://private-user-images.githubusercontent.com/122029652/380644128-11296fd5-d842-4171-9400-616f4aa98481.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAxMDczODYsIm5iZiI6MTczMDEwNzA4NiwicGF0aCI6Ii8xMjIwMjk2NTIvMzgwNjQ0MTI4LTExMjk2ZmQ1LWQ4NDItNDE3MS05NDAwLTYxNmY0YWE5ODQ4MS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyOFQwOTE4MDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MzhlZDU2OTBiNTIwMTRjMmE4Y2YxZjZhMWIwZjk0NGUxOTFmMGQ0NDI0NWU3MzRjNWRhZjYxMTdhMzAwM2NhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.LMGP535iyy7qq78cEmapymqjloI76xtevGhEvbiCAxs)

You're missing the `-m`, it's:
 `python.exe -m pip install -r requirements.txt`
 
 Also you need to point the command to use both files properly, run this in the `ComfyUI_windows_portable` -folder:
 
`python_embeded\python.exe -m pip install -r ComfyUI\custom_nodes\ComfyUI-VEnhancer\requirements.txt`

**Comment by kijai** [comment]
> Thanks Kijai, I still get this window.... ![Screenshot 2024-10-28 103523](https://private-user-images.githubusercontent.com/122029652/380655424-7b7c028c-39c1-47e6-9ec7-444e9e27943e.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAxMTE1MjcsIm5iZiI6MTczMDExMTIyNywicGF0aCI6Ii8xMjIwMjk2NTIvMzgwNjU1NDI0LTdiN2MwMjhjLTM5YzEtNDdlNi05ZWM3LTQ0NGU5ZTI3OTQzZS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyOFQxMDI3MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05OGRiNTc4YTc5OThmMjUyOWMzZmUxYzlhNDdhNDQ4Mjg2OTBlMjE2Yzk1NTg3OTU2ZDUwM2ExNzBlMmE1NmY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.PrIYtNcLuLjfFX4J__OUEcmGNO7puByJNXfscqGZz7U)

Maybe try running the command line with admin rights? 

![image](https://github.com/user-attachments/assets/f9842afc-d610-4525-a1b6-0f445a0aa3c8)

**Comment by yovsac** [comment]
hehehe, still getting the same message. Going to start crying any moment now. hehehe

**Comment by kijai** [comment]
It is a permission issue in your operating system, try moving the whole ComfyUI portable outside of the User folder maybe. You do have admin rights on that PC right?

**Comment by yovsac** [comment]
this is mega strange. Now my comfy doesnt work, get the same message. Even moving it to C drive have the same problem. I might have to reinstall comfy and all. Ill let you know...

**Comment by kijai** [comment]
> this is mega strange. Now my comfy doesnt work, get the same message. Even moving it to C drive have the same problem. I might have to reinstall comfy and all. Ill let you know...

Yeah since it's now allowing you to run the embedded python.exe, I was wondering how your Comfy could even work

**Comment by yovsac** [comment]
finally!!! don't know what happened. I had to reinstall comfy. Maybe some incompatibility with a custom node? who knows. Thanks again for your help and patience Kijai!!
